components/accelerator/nvidia/clock-speed/metrics.go

```diff
@@ -40,22 +34,7 @@ var (
 	).MustCurryWith(componentLabel)
 )
 
-var _ components.PromRegisterer = &component{}
-
-func (c *component) RegisterCollectors(reg *prometheus.Registry, dbRW *sql.DB, dbRO *sql.DB, tableName string) error {
-	if err := reg.Register(graphicsMHz); err != nil {
-		return err
-	}
-	if err := reg.Register(memoryMHz); err != nil {
-		return err
-	}
-
-	return nil
-}
-
-// TO BE DEPRECATED
-func (c *component) Metrics(ctx context.Context, since time.Time) ([]components.Metric, error) {
-	log.Logger.Debugw("querying metrics", "since", since)
-
-	return nil, nil
+func init() {
+	prometheus.MustRegister(metricGraphicsMHz)
+	prometheus.MustRegister(metricMemoryMHz)
```

Comment:
We should define a prometheus registry in metric package, each component should register metrics into it.
```suggestion
	prometheus.MustRegister(
	         metricGraphicsMHz, 
	         metricMemoryMHz,
	)
```

Reply:
done